x-pack/plugins/rollup/public/crud_app/services/track_user_action.js

```diff
@@ -13,8 +13,16 @@ export function trackUserAction(actionType) {
   getHttp().post(userActionUri);
 }
 
+/**
+ * Transparently return provided request Promise, while allowing us to track
+ * a successful completion of the request.
+ */
 export function trackUserRequest(request, actionType) {
   // Only track successful actions.
-  request.then(() => trackUserAction(actionType));
-  return request;
+  return request.then(response => {
```

Comment:
@bmcconaghy I updated the Rollups code based on some suggestions from @sebelga about how to make the code easier to follow. Do you find this clearer?